tee-worker/core-primitives/substrate-sgx/externalities/src/lib.rs

```diff
@@ -110,6 +110,12 @@ pub trait SgxExternalitiesTrait {
 	/// Get the next key in state after the given one (excluded) in lexicographic order.
 	fn next_storage_key(&self, key: &[u8]) -> Option<Vec<u8>>;
 
+	/// Iter prefix
+	fn iter_prefix<K: Decode + Debug, V: Decode + Debug>(
```

Comment:
This is low-level change, I'd highly recommend having a unit test for it if you want to add this fn

Reply:
Added a unit test covering this function ✅